dist/profile/templates/jenkinscontroller/casc/csp-plugin.yaml.erb

```
@@ -0,0 +1,10 @@
+<%- if @jcasc['csp-plugin'] && @jcasc['csp-plugin']['reportOnly'] -%>
```

your configuration is doing more than reportOnly so it shouldn't really be gated on that.

If you do want to keep it then the if block on L4-6 should be removed as this if block ensures its always set
```suggestion
<%- if @jcasc['csp-plugin'] -%>
```

---

you are perfectly right ! thank you @timja !